main/guides/orchestration/index.md

```diff
@@ -1,11 +1,20 @@
+# What is Agoric Orchestration?
+
+Agoric’s Orchestration capability allows developers to easily build cross-chain capabilities into existing applications or to create novel cross-chain-focused products.
+
+The Agoric Orchestration API sits on top of Agoric’s novel VM that provides three key elements that make multichain applications possible:
+
+- **Remote account control and transfer**: Use the Orchestration APIs to easily create accounts on remote chains, transfer assets, and invoke contracts. Your Agoric contract orchestrates all behavior directly.
+- **Multiblock execution with async & await**: Agoric applications communicate asynchronously and await responses which may come in the same block or many blocks (or weeks!) later. Contracts simply continue executing when the response arrives.
+- **On-chain Timers**: Contracts can set timers for regular execution which makes executing common activities like subscriptions easy.
+
+Agoric’s Orchestration API simplifies controlling remote chains, moving assets, and using capabilities on any chain the API reaches.
```

Comment:
**any chain the API reaches** sounds very informal - it's fine if marketing is onboard with it.